aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -49,7 +49,7 @@ void mul_kernel(TensorIterator& iter) {
 }
 
 void div_kernel(TensorIterator& iter) {
-  if (isIntegralType(iter.dtype())) {
+  if (isIntegralType(iter.dtype(), /*includeBool*/ false)) {
```

Comment:
div for bools wasn't enabled before and this change shouldn't enable it now

Reply:
I'm a bit confused on what's going on here -- shouldn't this not execute because AT_DISPATCH_INTEGRAL_TYPES doesn't include bool?

Reply:
It wont. It will give an error like "not implemented to CPUBool or something like that"

Reply:
ok, but that's different than your original comment which is about actually enabling bool.